### PR TITLE
Update App to bring to playground

### PR DESCRIPTION
**Pull request by T-Kukawka**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,11 +1,12 @@
 version: 2.1
 orbs:
-  architect: giantswarm/architect@0.8.5
+  architect: giantswarm/architect@0.16.0
```
Comment (glitchcrab) [inline .circleci/config.yml:3]: any reason why you don't want to use the latest? `1.1.1` is available.

File: helm/velero-app/values.yaml [hunk 2]
```diff
@@ -6,8 +6,7 @@
 # enabling restic). Required.
 image:
   repository: velero/velero
```
Comment (glitchcrab) [inline helm/velero-app/values.yaml:8]: do we want to retag the image and use quay?

File: .circleci/config.yml (hunk 1 above)
Comment (T-Kukawka) [reply .circleci/config.yml:3]: architect? did not know, based it on efk stack app

File: helm/velero-app/values.yaml (hunk 2 above)
Comment (T-Kukawka) [reply helm/velero-app/values.yaml:8]: don't know - do we? 😄

File: README.md [hunk 3]
```diff
@@ -59,10 +59,56 @@ credentials:
...
+    bucket: testing
+    config:
+      resourceGroup: r55p6
+      # Storage Account Name
+      storageAccount: testingveleroapp
+      bucket: testing
+
+initContainers:
+- name: velero-plugin-for-microsoft-azure
+      image: velero/velero-plugin-for-microsoft-azure:v1.1.1
```
Comment (pipo02mix) [inline README.md:80]: Can we update aws one to `v1.1.0` (latest) plz

File: .circleci/config.yml (hunk 1 above)
Comment (T-Kukawka) [reply .circleci/config.yml:3]: where i can check the latest for architect? i see version 3x smth in the repo

File: README.md (hunk 3 above)
Comment (T-Kukawka) [reply README.md:80]: not sure if i have time to test it but i can bump it 😄

File: .circleci/config.yml (hunk 1 above)
Comment (cokiengchiara) [reply .circleci/config.yml:3]: > Find the latest ARCHITECT_ORB_VERSION here: https://circleci.com/orbs/registry/orb/giantswarm/architect, then update the CircleCi config with it.

as described in https://intranet.giantswarm.io/docs/dev-and-releng/app-developer-processes/adding_app_to_appcatalog/ ;)

File: helm/velero-app/Chart.yaml [hunk 4]
```diff
@@ -1,4 +1,4 @@
-apiVersion: v1
+apiVersion: v2
 name: velero-app
```
Comment (cokiengchiara) [inline helm/velero-app/Chart.yaml:2]: From https://intranet.giantswarm.io/docs/dev-and-releng/app-developer-processes/adding_app_to_appcatalog/

> However, it is no longer required nor advisable to name your app and helm chart (step 2 & 3 below) with the -app suffix.

@ubergesundheit , this applies here, right? So better to name it `velero`?

File: helm/velero-app/values.yaml (hunk 2 above)
Comment (T-Kukawka) [reply helm/velero-app/values.yaml:8]: https://github.com/giantswarm/retagger/pull/539

**Comment by cokiengchiara** [comment]
1. FYI I did the repo stuff described in Step 1 here https://intranet.giantswarm.io/docs/dev-and-releng/app-developer-processes/adding_app_to_appcatalog/
2. Is this done?

Set executor: "app-build-suite" in the push-to-app-catalog CircleCI configuration (architect-orb docs). This packages your new app with app-build-suite, enabling metadata integration and packaging (Spec)

File: helm/velero-app/Chart.yaml (hunk 4 above)
Comment (ubergesundheit) [reply helm/velero-app/Chart.yaml:2]: if its a default app installed as part of a release, the `-app` suffix is still required

**Comment by ubergesundheit** [comment]
> Set executor: "app-build-suite" in the push-to-app-catalog CircleCI configuration (architect-orb docs). This packages your new app with app-build-suite, enabling metadata integration and packaging (Spec)

This also requires adding a `.abs` directory containing a `main.yaml` file with configuration. This one should be enough:

```
replace-chart-version-with-git: true
generate-metadata: true
chart-dir: ./helm/velero-app
destination: ./build
```

**Comment by T-Kukawka** [comment]
I followed principle from the ticket 'just get it available for customer' but i guess that is not enough 😅

File: helm/velero-app/Chart.yaml (hunk 4 above)
Comment (T-Kukawka) [reply helm/velero-app/Chart.yaml]: removed app suffix

File: .circleci/config.yml (hunk 1 above)
Comment (T-Kukawka) [reply .circleci/config.yml]: done

**Comment by cokiengchiara** [comment]
> I followed principle from the ticket 'just get it available for customer' but i guess that is not enough 😅

Yeah sorry about that @T-Kukawka .

**Review by ubergesundheit — APPROVED** [review]
Changes in circle config look good, no idea about the other stuff

**Review by cokiengchiara — APPROVED** [review]
LGTM if works for @T-Kukawka

**Comment by T-Kukawka** [comment]
waiting for rettagger, will test one more time and if works will merge/deploy it

**Comment by T-Kukawka** [comment]
aaaaand something broke...........

**Comment by T-Kukawka** [comment]
it somehow is taking old CM from somewhere i have no clue where...

**Comment by T-Kukawka** [comment]
seems like the resources leftovers
will try to clean up them in the morning and try again

**Comment by T-Kukawka** [comment]
okay downgrading to 1.4.3 as i found multiple issues on errors with the 1.5.3 that i am exactly facing, 1.4.3 works right away...

**Comment by T-Kukawka** [comment]
And this worked like a charm - my bad to upgrade to 1.5.3 in last test and getting mad as there were 'no visible changes' ....
MERGING

**Pull request merged**